client/state/selectors.js

```diff
@@ -38,7 +38,22 @@ const getRawFormErrors = ( schema, data ) => {
 
 export const getFormErrors = createSelector(
 	( state ) => state.form.errors,
-	( state, props ) => props.schema,
+	( state, schema ) => schema,
 	( state ) => state.settings,
 	( errors, schema, data ) => removeErrorDataPathRoot( errors || getRawFormErrors( schema, data ) )
 );
+
+export const getStepFormErrors = createSelector(
+	( state ) => state,
+	( state, schema ) => schema,
```

Comment:
I meant adding something like this befpre `getStepFormErrors` and `getFormErrors`

```
const getSchema = ( state, schema ) => schema;
```

and then, in `getStepFormErrors` and `getFormErrors` do something like this instead:

```
export const getStepFormErrors = createSelector(
	( state ) => state,
	getSchema,
```

IDK, its a small DRY - maybe too small